packages/mobile/src/screens/notifications-screen/Notifications/USDCPurchaseBuyerNotification.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback } from 'react'
+
+import type {
+  Nullable,
+  USDCPurchaseBuyerNotification as USDCPurchaseBuyerNotificationType,
+  TrackEntity
+} from '@audius/common'
+import { notificationsSelectors, Name } from '@audius/common'
+import { make } from 'audius-client/src/common/store/analytics/actions'
+import { useSelector } from 'react-redux'
+
+import IconCart from 'app/assets/images/iconCart.svg'
+import { useNotificationNavigation } from 'app/hooks/useNotificationNavigation'
+
+import {
+  NotificationTile,
+  NotificationHeader,
+  NotificationText,
+  NotificationTitle,
+  NotificationTwitterButton,
+  UserNameLink,
+  EntityLink
+} from '../Notification'
+
+const { getNotificationUsers, getNotificationEntity } = notificationsSelectors
+
+const messages = {
+  title: 'Purchase Successful',
+  youJustPurchased: 'You just purchased ',
+  from: ' from ',
+  exclamation: '!',
+  twitterShare: (
+    trackTitle: string,
+    sellerHandle: string,
+    trackPermalink: string
+  ) =>
+    `I bought the track ${trackTitle} by ${sellerHandle} on Audius! #AudiusPremium https://audius.co${trackPermalink}`
+}
+type USDCPurchaseBuyerNotificationProps = {
+  notification: USDCPurchaseBuyerNotificationType
+}
+
+export const USDCPurchaseBuyerNotification = (
+  props: USDCPurchaseBuyerNotificationProps
+) => {
+  const { notification } = props
+  const navigation = useNotificationNavigation()
+  const track = useSelector((state) =>
+    getNotificationEntity(state, notification)
+  ) as Nullable<TrackEntity>
```

Comment:
Would be nice if we could add typing directly to the selector in the future so we don't have to cast it :-(